src/components/app/details/appDetails/SourceInfo.tsx

```diff
@@ -231,7 +230,7 @@ export const SourceInfo = ({
                     {isVirtualEnvironment && renderGeneratedManifestDownloadCard()}
                     {!loadingResourceTree && (
                         <IssuesCard
-                            appStreamData={appStreamData}
+                            appConditions={appDetails.resourceTree?.conditions}
```

Comment:
We can remove this as we can extract the appDetails in IssuesCard

Reply:
done